rust/models/src/scan.rs

```diff
@@ -13,9 +13,9 @@ use super::{scanner_preference::ScannerPreference, target::Target, vt::VT};
 )]
 #[cfg_attr(feature = "bincode_support", derive(bincode::Encode, bincode::Decode))]
 pub struct Scan {
-    #[cfg_attr(feature = "serde_support", serde(default))]
+    #[cfg_attr(feature = "serde_support", serde(skip_deserializing, default = "uuid"))]
```

Comment:
I don't think we can do it that way as we use deserializing and serializing when using infisto. This means that everytime we deserialize we have a new scan_id.

Reply:
Yes, you are right, I forgot about it...

Reply:
maybe it is better to use `#[serde(default)]` and override the scan_id on post scan inside entry?